conf/available_modules.go

```diff
@@ -12,6 +12,7 @@ import (
 	_ "mig/modules/netstat"
 	_ "mig/modules/ping"
 	_ "mig/modules/pkg"
+	_ "mig/modules/pkgprint"
```

Comment:
Is the intent to limit this module to packages fingerprints? If not, would calling it `fingerprint` make more sense?